tests/phpunit/tests/test-strings.php

```diff
@@ -209,4 +210,60 @@ public function test_switch_to_locale() {
 
 		$GLOBALS['wp_locale_switcher'] = $old_locale_switcher; // Reset the original global var
 	}
+
+	public function test_retrieve_password_default_language() {
+		// Create string translations.
+		$_mo = new PLL_MO();
+		$_mo->add_entry( $_mo->make_entry( get_bloginfo( 'name' ), 'My Site' ) );
+		$_mo->export_to_db( self::$model->get_language( 'en' ) );
+
+		reset_phpmailer_instance();
+		$user_id = self::factory()->user->create(
+			array(
+				'user_login' => 'janeDoe',
+				'user_email' => 'jane.doe@example.com',
+				'locale'     => 'en_US',
+			)
+		);
+
+		// Set Polylang environment.
+		$admin = new PLL_Admin( $this->links_model );
+		$admin->init();
+
+		// Let's send the mail.
+		$mailer = tests_retrieve_phpmailer_instance();
+		$result = retrieve_password( 'janeDoe' );
```

Comment:
To make it compatible with WP < 5.7, you can set the user with `$_POST['user_login']`.

Reply:
Thanks ! Done in f055b58